data/actionsmap/yunohost.yml

```diff
@@ -535,6 +535,31 @@ app:
                     full: --file
                     help: Folder or tarball for upgrade
 
+        ### app_change_url()
+        change_url:
```

Comment:
Done.

Shouldn't we rename the script "change_url" to "changeurl" too to be consistent?